tools/logger/convert.c

```diff
@@ -218,8 +234,21 @@ static void process_params(struct proc_ldc_entry *pe,
 			memmove(&p[2], &p[uuid_fmt_len], (int)(t_end - &p[uuid_fmt_len]) + 1);
 			p += uuid_fmt_len - 2;
 			t_end -= uuid_fmt_len - 2;
+		} else if (p + 2 < t_end && p[1] == 'p' && p[2] == 'E') {
```

Comment:
CAn we have an inline comment here stating what we are checking for

Reply:
done